.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: python
+python:
+    - "2.7"
+    - "3.3"
+install:
+    pip install -r requirements.txt -e .
+script: nosetests
```

Comment:
Travis was complaining about the file not being executable. Decided to do this to unblock me.